MovieApp/PresentationLayer/MovieDetails/ReviewView/ReviewView+Design.swift

```diff
@@ -100,9 +102,23 @@ extension ReviewView: ConstructViewsProtocol {
         }
 
         reviewLabel.snp.makeConstraints {
-            $0.top.equalTo(headerView.snp.bottom).offset(15)
+            $0.top.equalTo(headerView.snp.bottom).offset(3 * spacing)
             $0.leading.trailing.bottom.equalToSuperview()
         }
     }
 
+    @objc
+    private func toggleExpandCollapse(tapGesture: UITapGestureRecognizer) {
+        if let label = tapGesture.view as? UILabel {
+            UIView.transition(with: label, duration: 0.2, options: .transitionCrossDissolve) {
+                if label.numberOfLines == self.numberOfLines {
+                    label.numberOfLines = 0
+                } else if label.numberOfLines == 0 {
+                    label.numberOfLines = self.numberOfLines
+                }
+            }
+            self.expandCollapseReview()
```

Comment:
should this be in animate block?

Reply:
Putting it in the animate block results in the label being animated while it is being laid out (it drops from the top of the containing view's frame).
`expandCollapseReview` is a bit of a misnomer given that it actually just changes the text in the label (which I'm assuming results in the label being laid out again resulting in the previously mentioned behavior).